pkg/data/components/LogDeduplicationProcessor.yaml

```diff
@@ -22,7 +22,8 @@ ports:
 properties:
   - name: Interval
     type: duration
-    validations: [duration]
+    validations:
```

Comment:
this makes me wonder if we could leave validations empty and use the type for validation in such cases

Reply:
Type isn't always sufficient. Normal type validations, yes -- which is why there are no validations like "int" or "bool". But range, or limited numbers of strings need more flexibility